content/bd/0004-relational-model-conversion.md

```diff
@@ -74,8 +73,8 @@ $$
 
 Para uma relação, podemos determinar:
 
-- O [**grau de uma relação**](color:orange) é o número de atributos (ou se preferirem, campos ou colunas)
-- A [**cardinalidade de uma relação**](color:yellow) é o número de tuplos (isto é, linhas)
+- O [**grau de uma relação**](color:orange), que corresponde ao número de atributos (ou se preferirem, campos ou colunas)
+- A [**cardinalidade de uma relação**](color:yellow), que equivale ao número de tuplos (isto é, linhas)
 
 Uma relação é um [**objeto matemático**](color:purple) que é representável como uma tabela.
```

Comment:
```suggestion
Uma relação é um [**objeto matemático**](color:purple) que é **representável** como uma tabela.
```
Add emphasis. Professors made it clear that, formally, a relation is not a table - a table is simply a representation of a relation, despite the terms being used interchangeably in practice.